framework/src/base/SubProblem.C

```diff
@@ -342,3 +343,12 @@ SubProblem::checkMatProps(std::map<unsigned int, std::set<std::string> > & props
       mooseError("No material defined on " + type + " " + check_name);
   }
 }
+
+unsigned int
+SubProblem::getAxisymmetricRadialCoord()
+{
+  if (_rz_coord_axis == 0)
+    return 1;
+  else
+    return 0;
+}
```

Comment:
I know there's a short description of this method in the header but the resulting implementation is just screwy. A short reason here might help too as to why these are inverted.